main/include/rotate45.h

```diff
@@ -1,12 +1,26 @@
 #ifndef ROTATE45_H
 #define ROTATE45_H
 #include "algo.h"
+#include <cmath>
 enum Direction45 {CLOCKWISE45,COUNTERCLOCKWISE45};
 class Rotate45: public Algorithm {
 public:
     QImage processImage(const QImage *workingModel);
     friend class Rotate45Builder;
 private:
+    QImage& blur_left_upper_angle(QImage& img,
```

Comment:
```suggestion
    QImage& blur_left_upper_angle(QImage& img,
```
lowerCamelCase а не снейк пожалуйста в названиях